### PR TITLE
gather must-gather data after we run tests

### PR DESCRIPTION
**Pull request by osherdp**
Right now the must-gather data we have is pre-tests. We want to do the actual gathering afterwards in order to investigate failures of tests.
/cc @YuviGold @omertuc 
/hold

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/sno/gather/baremetalds-sno-gather-commands.sh [hunk 1]
```diff
@@ -31,4 +31,11 @@ sosreport --ticket-number "\$HOSTNAME" --batch -o container_log,filesys,kvm,libv
 
 cp -r /home/sno/build/ /tmp/artifacts/
 
+must_gather_dir=/tmp/artifacts/must-gather
```
[the remark this answers is not in the document]
Comment (osherdp) [reply ci-operator/step-registry/baremetalds/sno/gather/baremetalds-sno-gather-commands.sh]: take a look at the new name

**Comment by omertuc** [comment]
👌

/test pj-rehearse

File: ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ fi
 # TODO: Figure out way to get these parameters (used by deploy_ibip) without hardcoding them here
 # preferrably by making deploy_ibip / makefile perform these configurations itself in the assisted_test_infra
 # repo.
-export SINGLE_NODE_IP_ADDRESS="192.168.126.10"
+export SINGLE_NODE_IP_ADDRESS="192.168.127.10"
```
Comment (YuviGold) [inline ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh:40]: Anyway to make it not hardcoded?

Comment (osherdp) [reply ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh:40]: open a ticket and assign on @omertuc

Comment (YuviGold) [reply ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh:40]: https://issues.redhat.com/browse/MGMT-6803

**Comment by omertuc** [comment]
Thank you, looks great

/lgtm

**Pull request merged**